src/main/java/org/pac4j/vertx/verticle/DemoServerVerticle.java

```diff
@@ -121,7 +121,13 @@ public void start() {
         // Requires authentication endpoint without specific authenticator attached
         addProtectedEndpointWithoutAuthorizer("/protected/index.html", "", router);
 
-        router.get("/index.html").handler(DemoHandlers.indexHandler(config));
+        // Direct basic auth authentication (web services)
+        addProtectedEndpointWithoutAuthorizer("/dba/index.html", "DirectBasicAuthClient", router);
```

Comment:
OK. Thanks for the explanations. There is no security issue with saving the user in session even for a web service call, it is just not necessary. Let vertx-web rule on this and pac4j follow the pace.